app/javascript/components/Checkout/TemporaryLibrary.tsx

```diff
@@ -103,25 +104,27 @@ const Card = ({
   nativeType: ProductNativeType;
   creator: Creator | null;
 }) => (
-  <article className="product-card relative">
-    <figure>
-      <Thumbnail url={thumbnailUrl} nativeType={nativeType} />
-    </figure>
-    <header>
+  <ProductCard>
+    <ProductCardFigure>
+      <Thumbnail url={thumbnailUrl} nativeType={nativeType} className="h-full w-full object-cover" />
```

Comment:
These `h-full w-full object-cover` styles are duplicated a lot ... this might be a rare case where we should use descendant selectors, i.e. add `[&_img]:size-full` etc to the figure classes